e2e/support/helpers/e2e-notebook-helpers.ts

```diff
@@ -32,6 +32,15 @@ export function getNotebookStep(
   return cy.findByTestId(`step-${type}-${stage}-${index}`);
 }
 
+export function getStepPreviewButton(
+  type: Exclude<NotebookStepType, "aggregate" | "breakout">,
+  { stage = 0, index = 0 } = {},
+): Cypress.Chainable<JQuery<HTMLElement>> {
+  return getNotebookStep(type, { stage, index }).findByTestId(
+    "step-preview-button",
+  );
```

Comment:
nit: no test asserts that the "eye" icon is used :)
Of course the name does not guarantee that correct svg is used, but it's better than nothing.

```suggestion
  return getNotebookStep(type, { stage, index }).icon("eye");
```